README.md

```diff
@@ -89,25 +83,46 @@ fpm run \
   -- --network model.json
 
 ```
-where `model.json` must be a neural network in the [JSON] format used by Inference-Engine and the companion [nexport] package.
-Automatic parallelization for training is under development.
+where `model.json` must be a neural network in the [JSON] format used by Fiats and the companion [nexport] package.
+
+Automatic parallelization for training neural networks is under development.
+
+#### Partially Supported Compilers
 
-#### Intel (`ifx`)
+Fiats release 0.14.0 and earlier support the use of the NAG, GNU, and Intel Fortran compilers.
+We are corresponding with these compilers' developers about addressing the compiler issues preventing building newer Fiats releases.
+
+##### NAG (`nagfor`)
+```
+fpm test --compiler nagfor --flag -fpp --profile release
+```
+
+##### GNU (`gfortran`)
+Compiler bugs related to parameterized derived types currently prevent `gfortran` from building Fiats versions 0.15.0 or later.
+Test and build earlier versions of Fiats build with the following command:
+```
+fpm test --compiler gfortran --profile release
+```
+
+##### Intel (`ifx`)
+Compiler bugs related to parameterized derived types currently prevent `gfortran` from building Fiats versions 0.15.0 or later.
```

Comment:
This line reference gfortran but is talking about ifx. Is the below suggestion sufficient to be accurate to the current situation?

```suggestion
Compiler bugs related to parameterized derived types currently prevent `ifx` from building Fiats versions 0.15.0 or later.
```

Reply:
```suggestion
Compiler bugs related to generic name resolution currently prevent `ifx` from building Fiats versions 0.15.0 or later.
```